plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveSessionProperties.java

```diff
@@ -338,6 +339,23 @@ public HiveSessionProperties(
                         "Parquet: Maximum number of rows passed to the writer in each batch",
                         parquetWriterConfig.getBatchSize(),
                         false),
+                new PropertyMetadata<>(
```

Comment:
nit: We should most likely move Parquet session properties to a different `SessionPropertiesProvider` for other connectors to use

Reply:
Unrelated, please file a separate GH issue for it